containers/datalab/Dockerfile.in

```diff
@@ -50,7 +50,7 @@ RUN pip install -U tornado==4.3 \
                    argparse==1.2.1 \
                    mock==1.2.0 \
                    requests==2.9.1 \
-                   oauth2client==1.5.2 \
+                   oauth2client==2.0.2 \
```

Comment:
This could be blocking for this pull request. The `gcloud` package requires `oauth2client >= 2.0.1`.

This seems to be related to the [switch from `SignedJwtAssertionCredentials` to `ServiceAccountCredentials` in `oauth2client 2.0.0`](https://github.com/google/oauth2client/blob/e99a4ac3c5d3c2944680ea180b71a0d144d6128b/CHANGELOG.md#special-note-about-serviceaccountcredentials).

Does that actually affect `tensorflow`, or just `dataflow`? It doesn't look to me like the `tensorflow` [`setup.py`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py) requires oauth2client.

@dhermes Any thoughts?